app/api/v1/scoring.py

```
@@ -0,0 +1,135 @@
+from fastapi import APIRouter, Depends, HTTPException, Query
```

Ruff (0.12.2)

`fastapi.Depends` imported but unused

Remove unused import: `fastapi.Depends`

(F401)